bitcoin/src/blockdata/script/borrowed.rs

```diff
@@ -218,6 +218,59 @@ impl Script {
         }
     }
 
+    /// Checks whether a script pubkey is a bare multisig output.
+    ///
+    /// In a bare multisig pubkey script the keys are not hashed, the script
+    /// is of the form:
+    ///
+    ///    2 &lt;pubkey1&gt; &lt;pubkey2&gt; &lt;pubkey3&gt; 3 OP_CHECKMULTISIG
```

Comment:
```suggestion
    ///    `2 <pubkey1> <pubkey2> <pubkey3> 3 OP_CHECKMULTISIG`
```

Yours works but its hard on the eyes for those of use who read the docs inline in the code.

Reply:
My bad, I could have suggested this last time and saved you the extra work.

Reply:
Oof, yeah.

Reply:
36138f8fc495f0e057d470d132def964849990f9